### PR TITLE
Tweak teleporter success message

### PR DESCRIPTION
**Pull request by DL6ER**
# What does this implement/fix?

Address comment by @yubiuser at https://github.com/pi-hole/web/pull/2738#pullrequestreview-1683980757

**Related issue or feature (if applicable):** N/A

**Pull request in [docs](https://github.com/pi-hole/docs) with documentation (if applicable):** N/A

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

## Checklist:

- [x] The code change is tested and works locally.
- [x] I based my code and PRs against the repositories `developmental` branch.
- [x] I [signed off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions
- [x] I [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) all my commits. Pi-hole requires signatures to verify authorship
- [x] I have read the above and my PR is ready for review.

### CODE REVIEW

**Review by yubiuser — DISMISSED** [review]

File: scripts/pi-hole/js/settings-teleporter.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ function importZIP() {
         if (data.error.hint !== null) $("#modal-import-error-message").text(data.error.hint);
       } else if ("files" in data) {
         $("#modal-import-success").show();
-        $("#modal-import-success-title").text("Import successful");
+        $("#modal-import-success-title").text(
+          "Import successful, please run pihole -g to update gravity."
```
Comment (PromoFaux) [inline scripts/pi-hole/js/settings-teleporter.js]: Maybe a hyperlink to the web interface's own gravity update page?

Comment (rdwebdesign) [reply scripts/pi-hole/js/settings-teleporter.js]: Just throwing ideas around:

Use a button or move the text+link from the alert title

<details>
<summary>View examples</summary>

image

image

</details>

Comment (yubiuser) [reply scripts/pi-hole/js/settings-teleporter.js]: I like the idea to move the text to the bottom. The button alone does not indicate that users should run gravity.

Comment (DL6ER) [reply scripts/pi-hole/js/settings-teleporter.js]: How about...

Screenshot from 2023-11-03 18-50-46

Comment (rdwebdesign) [reply scripts/pi-hole/js/settings-teleporter.js]: I think it's good.

**Review by yubiuser — APPROVED** [review]

**Review by rdwebdesign — APPROVED** [review]
Approved.

I won't merge it because I'm not sure if someone else wants to request another change.

**Pull request merged**